ibis/common/graph.py

```diff
@@ -284,17 +264,19 @@ def __init__(self, mapping=(), /, **kwargs):
         super().__init__(mapping, **kwargs)
 
     @classmethod
-    def from_bfs(cls, root: Node, filter=Node) -> Self:
+    def from_bfs(cls, root: Node, filter: Optional[Any] = None) -> Self:
         """Construct a graph from a root node using a breadth-first search.
 
         The traversal is implemented in an iterative fashion using a queue.
 
         Parameters
         ----------
-        root : Node
+        root
             Root node of the graph.
-        filter : Optional[type], default None
-            Type to filter out for the traversal, Node is filtered out by default.
+        filter
+            Pattern-like object to filter out nodes from the traversal. Essentially
```

Comment:
Not a huge fan of `Essentially` here :)

Is it necessary? It seems like the sentence stands on its own with `Essentially`.

Reply:
Nope, removing.